README.md

```diff
@@ -1001,42 +1001,41 @@ attacks*).
 
 ### <a name="codelyzer-rules"></a> Codelyzer rules
 ```json
-"banana-in-box": true,
-"contextual-life-cycle": true,
-"decorator-not-allowed": true,
-"pipe-impure": true,
-"templates-no-negated-async": true,
-"trackBy-function": true,
-"i18n": [
+"template-banana-in-box": true,
+"contextual-lifecycle": true,
+"contextual-decorator": true,
+"no-pipe-impure": true,
+"template-no-negated-async": true,
+"template-i18n": [
   true,
   "check-id",
   "check-text"
 ],
-"no-attribute-parameter-decorator": true,
+"component-max-inline-declarations": true,
+"no-attribute-decorator": true,
+"no-conflicting-lifecycle": true,
 "no-forward-ref": true,
 "no-input-rename": true,
-"no-life-cycle-call": true,
-"no-output-named-after-standard-event": true,
+"no-lifecycle-call": true,
+"no-output-native": true,
 "no-output-on-prefix": true,
 "no-output-rename": true,
-"no-queries-parameter": true,
 "no-template-call-expression": true,
```

Comment:
I'm not the repo owner, but I think this is supposed to get changed to `template-no-call-expression`.

See `Breaking Changes` here: https://github.com/mgechev/codelyzer/blob/master/CHANGELOG.md#500-2019-03-27

It also looks like 
- `no-use-before-declare` is deprecated
- `angular-whitespace` (below) is deprecated with the message `Use a formatter like Prettier for formatting purposes`.

Reply:
I think you mean `"no-queries-parameter"` should be changed to `"no-queries-metadata-property"`, not `"template-no-call-expression".

According to the TSLint docs [`no-use-before-declare`](https://palantir.github.io/tslint/rules/no-use-before-declare/) is not deprecated.

I agree to remove `angular-whitespace` (see https://github.com/mgechev/codelyzer/pull/733):

> Since angular-whitespace was deprecated and has known issues that won't be fixed, we should not
recommend it in a default configuration.